### PR TITLE
fleet: add marathon, have chronos require it

### PR DESCRIPTION
**Pull request by chr0n1x**

### CODE REVIEW

**Comment by iderdik** [comment]
:+1:

File: v2/fleet/03-marathon.service [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Environment=MARATHON_IMAGE=mesosphere/marathon:v0.10.0
+Environment=ZOOKEEPER=localhost:2181
+
+User=core
+Restart=always
+RestartSec=20
+TimeoutStartSec=0
+ExecStartPre=-/usr/bin/docker kill marathon
+ExecStartPre=-/usr/bin/docker rm marathon
+ExecStartPre=/usr/bin/docker pull ${MARATHON_IMAGE}
```
Comment (cfortier2) [inline v2/fleet/03-marathon.service]: lets move the pull before the kill, in case docker pull fails for some reason.

**Comment by cfortier2** [comment]
I'm not sure we need to add numbers to the file names. systemd ordering should handle it.

**Comment by chr0n1x** [comment]
@cfortier2 the README acknowledges this.
I just implemented it because I wanted a quick way to see when each unit is run.

**Comment by chr0n1x** [comment]
I _can_ revert it though

**Comment by cfortier2** [comment]
it's fine as is, and if it makes life easier for you then keep it. I'm just being whiney because its 3 more characters to type whenever we need to do anything with it :)

**Comment by chr0n1x** [comment]
I'll remove it. Because, you know, tab-completion isn't a thing

**Pull request merged**